doc/API.md

```diff
@@ -1118,23 +1160,25 @@ Once you have a libp2p instance, you can listen to several events it emits, so t
 If `autoDial` option is `true`, applications should **not** attempt to connect to the peer
 unless they are performing a specific action. See [peer discovery and auto dial](./PEER_DISCOVERY.md) for more information.
 
-- `peer`: instance of [`PeerInfo`][peer-info]
+- `peer`: instance of [`PeerId`][peer-id]
+
+### libp2p.connectionManager
 
 #### A new connection to a peer has been opened
 
 This event will be triggered anytime a new Connection is established to another peer.
 
 `libp2p.on('peer:connect', (peer) => {})`
 
-- `peer`: instance of [`PeerInfo`][peer-info]
+- `peer`: instance of [`PeerId`][peer-id]
```

Comment:
We should change this to connection per https://github.com/libp2p/js-libp2p/pull/610#discussion_r414466296

Reply:
Done!